akka-http-core/src/main/scala/akka/http/impl/util/JavaMappingCore.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2009-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.http.impl.util
+
+import akka.annotation.InternalApi
+import akka.http.javadsl.{ ConnectionContext, HttpConnectionContext, HttpsConnectionContext, settings ⇒ js }
+import akka.http.{ javadsl ⇒ jdsl, scaladsl ⇒ sdsl }
+
+/** INTERNAL API */
+@InternalApi
+private[http] object JavaMappingCore extends JavaMappings {
```

Comment:
To avoid moving all those classes to the `akka-http-model` module, I split the `JavaMapping` into 2 different files.